lib/app/views/default.erb

```diff
@@ -226,7 +226,39 @@
               label: 'Group ID',
               instance: instance,
               value: instance.group_id,
-              })%>                 
+              })%>   
+        <%= erb(:prerequisite_field,{},{prerequisite: :bulk_token_endpoint,
+              label: 'Bulk Token Endpoint',
+              instance: instance,
+              value: instance.bulk_token_endpoint,
+              })%>                
+        <%= erb(:prerequisite_field,{},{prerequisite: :bulk_client_id,
+              label: 'Bulk Client ID',
+              instance: instance,
+              value: instance.bulk_client_id,
+              })%>                             
+        <div class="form-group" data-requiredby="<%= test_set.variable_required_by(:bulk_public_key).map {|seq| seq.sequence_name}.join(',')%>" data-definedby="" data-prerequisite="bulk_public_key">
+
+              <label for="bulk_public_key">Bulk Public Key
+                  <small>
+                          If you have provided your JWKS at registration time, please enter the public key here. Leave it blank if the keys have not
+                          been exchanged ahead of time.
+                        </small>
+
+              </label>
+              <textarea type="text" class="form-control" name="bulk_public_key" id="bulk_public_key" placeholder="" rows="5"><%=instance.bulk_public_key %></textarea>
+        </div>
+        <div class="form-group" data-requiredby="<%= test_set.variable_required_by(:bulk_private_key).map {|seq| seq.sequence_name}.join(',')%>" data-definedby="" data-prerequisite="bulk_private_key">
+
+              <label for="bulk_private_key">Bulk Private Key
```

Comment:
This file needs to be rebased before it can be really reviewed.  @arscan added some HTML elements with similar element ids that may need deconfliction (e.g. `bulk_private_key`)

https://github.com/onc-healthit/inferno/blob/c426caeb4ae413149ff84b9218e2ed5939c1eb6c/lib/app/views/default.erb#L330-L340